var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -62,47 +72,53 @@ def install(self, spec, prefix):
             '-DCMAKE_Fortran_FLAGS=-fPIC',
             '-DEXAMPLES_ENABLE=ON',
             '-DEXAMPLES_INSTALL=ON',
-            '-DFCMIX_ENABLE=ON'
+            '-DFCMIX_ENABLE=ON',
+            '-DMPI_ENABLE=%s' % self.on_off('+mpi'),
+            '-DLAPACK_ENABLE=%s' % self.on_off('+lapack'),
+            '-DKLU_ENABLE=%s' % self.on_off('+klu'),
+            '-DHYPRE_ENABLE=%s' % self.on_off('+hypre'),
+            '-DSUPERLUMT_ENABLE=%s' % self.on_off('+superlu'),
+            '-DOPENMP_ENABLE=%s' % self.on_off('+openmp'),
+            '-DPTHREAD_ENABLE=%s' % self.on_off('+pthread')
```

Comment:
I think most C/C++ and even Python coders are far more familiar with printf-style string substituion mechanisms. The .format mechanism is, IMHO, far less intuitive and unless there is a big reason the printf-style cannot be used, I don't see the value in it. In addition, it requires more characters which leads to longer lines and greater likelihood of having to split those lines for sake of flake8 compliance. I would rather change .format lines to %s substitutions. Thoughts?